integration_tests/build.rs

```diff
@@ -0,0 +1,84 @@
+use std::{
+  env,
+  fs::File,
+  io::Write,
+  path::{Path, PathBuf},
+};
+use quote::{format_ident, quote};
+use rustfmt_wrapper::rustfmt;
+
+type DynResult<T> = std::result::Result<T, Box<dyn std::error::Error>>;
+
+fn parse_file(path: &Path) -> DynResult<syn::File> {
+  let content = std::fs::read_to_string(path)?;
+  syn::parse_file(&content).map_err(|e| e.into())
+}
+
+fn main() -> DynResult<()> {
+  let out_dir = env::var_os("OUT_DIR").unwrap();
+  let manifest_dir = env::var("CARGO_MANIFEST_DIR")?;
+  let dest_path = PathBuf::from(&out_dir).join("generated_tests.rs");
```

Comment:
Thinking about it, env variables is not the right place to put it. I just would like it better if we defined the file name somewhere and used that define instead of hard coding the path. The goal was not to make it customizable, using cargo was just the  only way i could see us doing it.

Its also just a minor nit 😁